submissions/akulahere/task-js-post-oop/index.js

```diff
@@ -0,0 +1,76 @@
+class Inhabitant {
+  constructor(species, name, gender, legs, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.saying = saying;
+    this.friends = 'No friends' ;
+  }
+
+  makeFriend(friend) {
+    this.friends = this.friends === 'No friends' ? [] : this.friends;
```

Comment:
What if a peer developer wants to have e.g. 'none' instead of 'No friends'?
At least this should be a constant.
Not a really good approach to have interpretation of state instead of state.
Having empty array at initialization and interpretation at the point when we need this would be a more constistent approach. 
Why that? We may want to interpet the list of friends in a different way, e.g. having a count of friends.
This means that the logic inside `toString` becomes more complex (e.g. addint interim `.map()`), but it will a single point of complexity.